<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>donneurs</class>
 <widget class="QDialog" name="donneurs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1479</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1321</width>
     <height>641</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;a&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>27</height>
    </size>
   </property>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Ajouter Donneur</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>172</width>
       <height>551</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>id</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Nom du donneur</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Prenom du donneur</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>numero du telephone</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Adresse</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Date</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>601</width>
       <height>611</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLineEdit" name="id"/>
      </item>
      <item>
       <widget class="QLineEdit" name="nom"/>
      </item>
      <item>
       <widget class="QLineEdit" name="prenom"/>
      </item>
      <item>
       <widget class="QLineEdit" name="tel"/>
      </item>
      <item>
       <widget class="QLineEdit" name="adresse"/>
      </item>
      <item>
       <widget class="QDateEdit" name="date"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>990</x>
       <y>120</y>
       <width>221</width>
       <height>321</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QPushButton" name="pushButton_5">
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_6">
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_7">
        <property name="text">
         <string>Annuler</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Afficher la liste des Donneurs</string>
    </attribute>
    <widget class="QScrollBar" name="verticalScrollBar_2">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>60</y>
       <width>21</width>
       <height>541</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>-120</x>
       <y>200</y>
       <width>701</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>1040</x>
       <y>270</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>571</width>
       <height>611</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>10</y>
       <width>243</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2"/>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>80</y>
       <width>330</width>
       <height>28</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QCheckBox" name="checkBox_4">
        <property name="text">
         <string>Tri par nom</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_5">
        <property name="text">
         <string>Tri par prenom</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_6">
        <property name="text">
         <string>Tri par date</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>150</y>
       <width>172</width>
       <height>371</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_33">
        <property name="text">
         <string>id</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_26">
        <property name="text">
         <string>Nom du mat√©riel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_28">
        <property name="text">
         <string>prenom</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_30">
        <property name="text">
         <string>tel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_27">
        <property name="text">
         <string>adresse</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_29">
        <property name="text">
         <string>date de naissance</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>140</y>
       <width>127</width>
       <height>381</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="id4">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nom4"/>
      </item>
      <item>
       <widget class="QLineEdit" name="pre4"/>
      </item>
      <item>
       <widget class="QLineEdit" name="tel4"/>
      </item>
      <item>
       <widget class="QLineEdit" name="adr4"/>
      </item>
      <item>
       <widget class="QDateEdit" name="date4"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>593</x>
       <y>10</y>
       <width>214</width>
       <height>30</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Rechercher :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="recherche"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>1040</x>
       <y>160</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_10">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::PreventContextMenu</enum>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <attribute name="title">
     <string>Donneur de lan </string>
    </attribute>
    <widget class="QLabel" name="label_31">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>A :</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
